starter project not working (Helidon MP/Oracle/EclipseLink/Docker)

## Environment Details
* Helidon Version: 3.0.1
* Helidon MP
* JDK version:
* OS: Linux
* Docker version (if applicable):
----------

## Problem Description
Project starter does not compile

## Steps to reproduce
Use the starter and choose the 
- Helidon MP
- EclipseLink
- Oracle

Download the file.
./docker build .

..
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  16.278 s
[INFO] Finished at: 2022-10-19T05:54:42Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:java (weave) on project myproject: An exception occured while executing the Java class. 
[ERROR] Exception Description: An exception was thrown while weaving: [Ljava.lang.String;@99c074d
[ERROR] Internal Exception: Exception [EclipseLink-30009] (Eclipse Persistence Services - 3.0.3.v202208191135): org.eclipse.persistence.exceptions.PersistenceUnitLoadingException
[ERROR] Exception Description: An exception was thrown while trying to load persistence unit at url: file:/helidon/target/classes/
[ERROR] Internal Exception: Exception [EclipseLink-30004] (Eclipse Persistence Services - 3.0.3.v202208191135): org.eclipse.persistence.exceptions.PersistenceUnitLoadingException
[ERROR] Exception Description: An exception was thrown while processing persistence.xml from URL: file:/helidon/target/classes/
[ERROR] Internal Exception: 
[ERROR] (1. : XML-20220: (Fatal Error) Invalid InputSource.)

Note, mvn package works.

--- a/archetypes/helidon/src/main/archetype/mp/custom/database.xml
+++ b/archetypes/helidon/src/main/archetype/mp/custom/database.xml
@@ -130,6 +130,7 @@
                                 <argument>${project.build.outputDirectory}</argument>
                                 <argument>${project.build.outputDirectory}</argument>
                             </arguments>
+                            <skip>${eclipselink.weave.skip}</skip>
                         </configuration>
                     </execution>
                 </executions>

